app/src/main/java/org/sagebionetworks/research/mpower/tracking/TrackingMenuFragment.java

```diff
@@ -145,16 +138,14 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AndroidSupportInjection.inject(this);
         showTracking = getResources().getBoolean(R.bool.show_data_tracking);
         trackingMenuViewModel =
-                ViewModelProviders.of(this).get(TrackingMenuFragmentViewModel.class);
-        if (showTracking) {
```

Comment:
This was a separate bug where setting the selected tab based on a boolean every time in on create was ignoring the purpose of the ViewModel to save the tracking tab selection.